cli/src/main/java/com/devonfw/tools/ide/property/Property.java

```diff
@@ -231,9 +231,7 @@ protected String format(V valueToFormat) {
    */
   public void setValue(V value) {
 
-    if (!this.multivalued) {
-      this.value.clear();
-    }
+    this.value.clear();
```

Comment:
I was under the impression that for multi-valued arguments addValue instead of this setValue method would have been used, but I was mistaken.

Reply:
I have removed the clear call from the setValue method altogether, since it simply overwrites an existing value (that should not exist). Since now we clear all property values before running the commandlet, it serves no purpose. 
For proper handling, one might add an exception catch to this method or let the validate method handle multiple values set on a single-valued property, but this would be part of  #360.